wsi/drm/commondrmutils.h

```diff
@@ -31,50 +31,50 @@ static size_t drm_bo_get_num_planes(uint32_t format) {
     case DRM_FORMAT_ARGB8888:
     case DRM_FORMAT_AYUV:
     case DRM_FORMAT_BGR233:
-    case DRM_FORMAT_BGR565:
-    case DRM_FORMAT_BGR888:
     case DRM_FORMAT_BGRA1010102:
     case DRM_FORMAT_BGRA4444:
     case DRM_FORMAT_BGRA5551:
     case DRM_FORMAT_BGRA8888:
     case DRM_FORMAT_BGRX1010102:
     case DRM_FORMAT_BGRX4444:
     case DRM_FORMAT_BGRX5551:
-    case DRM_FORMAT_BGRX8888:
     case DRM_FORMAT_C8:
     case DRM_FORMAT_GR88:
     case DRM_FORMAT_R8:
     case DRM_FORMAT_RG88:
     case DRM_FORMAT_RGB332:
-    case DRM_FORMAT_RGB565:
-    case DRM_FORMAT_RGB888:
     case DRM_FORMAT_RGBA1010102:
     case DRM_FORMAT_RGBA4444:
     case DRM_FORMAT_RGBA5551:
     case DRM_FORMAT_RGBA8888:
     case DRM_FORMAT_RGBX1010102:
     case DRM_FORMAT_RGBX4444:
     case DRM_FORMAT_RGBX5551:
-    case DRM_FORMAT_RGBX8888:
     case DRM_FORMAT_UYVY:
     case DRM_FORMAT_VYUY:
     case DRM_FORMAT_XBGR1555:
     case DRM_FORMAT_XBGR2101010:
     case DRM_FORMAT_XBGR4444:
-    case DRM_FORMAT_XBGR8888:
     case DRM_FORMAT_XRGB1555:
     case DRM_FORMAT_XRGB2101010:
     case DRM_FORMAT_XRGB4444:
-    case DRM_FORMAT_XRGB8888:
     case DRM_FORMAT_YUYV:
     case DRM_FORMAT_YVYU:
     case DRM_FORMAT_R16:
       return 1;
+    case DRM_FORMAT_BGR565:
```

Comment:
hmm not sure I understand the change :)

Reply:
https://github.com/torvalds/linux/blob/master/include/uapi/drm/drm_fourcc.h#L116

These format modifiers are in fact 2 plane, and were incorrectly placed in the section returning 1.

Reply:
Ha, nice !

Reply:
:)

Reply:
I don't see any RGB + A formats in here, note the _A8 suffix.

Reply:
@strassek https://github.com/torvalds/linux/blob/master/include/uapi/drm/drm_fourcc.h#L118

Reply:
im gonna close this PR --- cause its all kinds of wrong.